fastlane_core/lib/fastlane_core/command_executor.rb

```diff
@@ -47,6 +47,8 @@ def execute(command: nil, print_all: false, print_command: true, error: nil, pre
           UI.command_output(loading)
         end
 
+        return command unless Helper.sh_enabled? # only for the tests
```

Comment:
I know the `sh` action works like this but I would like to keep any `Helper.sh_enabled?` outside of main code as much as possible (and since `FastlaneCommand::Executor` isn't `sh`) 😇 

So for now I think the best solution is here to probably do something like this in your tests...
```rb
before do
  # Force FastlaneCore::CommandExecutor.execute to return command
  allow(FastlaneCore::CommandExecutor).to receive(:execute).and_wrap_original do |m, *args|
    args[0][:command]
  end
end
```

Reply:
@joshdholtz Fixed. Your code worked perfectly! Thank you for the good solution 🙇